src/main/resources/META-INF/plugin.xml

```diff
@@ -5,6 +5,13 @@
     <depends>com.intellij.modules.platform</depends>
     <depends>com.intellij.modules.lang</depends>
     <depends optional="true" config-file="plugin-java.xml">com.intellij.modules.java</depends>
+    <depends optional="true" config-file="plugin-python.xml">com.intellij.modules.python</depends>
+<!--    TODO-->
+<!--    <depends optional="true" config-file="plugin-js.xml">JavaScript</depends>-->
```

Comment:
These would be needed to implement a `LanguagePsiParser` for e.g. JavaScript, PHP, ...
In the `plugin-js.xml` a `JavaScriptPsiParser` service would be added, which would only be loaded if the user uses IntelliJ Ultimate or WebStorm (which have JS support)